common/src/test/java/org/apache/sedona/common/raster/RasterBandAccessorsTest.java

```diff
@@ -75,6 +81,113 @@ public void testBandNoDataValueIllegalBand() throws FactoryException, IOExceptio
         assertEquals("Provided band index 2 is not present in the raster", exception.getMessage());
     }
 
+    @Test
+    public void testZonalStats() throws FactoryException, ParseException, IOException {
+        GridCoverage2D raster = rasterFromGeoTiff(resourceFolder + "raster_geotiff_color/FAA_UTM18N_NAD83.tif");
+        String polygon = "POLYGON ((236722 4204770, 243900 4204770, 243900 4197590, 221170 4197590, 236722 4204770))";
+        Geometry geom = Constructors.geomFromWKT(polygon, RasterAccessors.srid(raster));
+
+        double actual = RasterBandAccessors.getZonalStats(raster, geom, 1, "sum", false);
+        double expected = 1.0690406E7;
+        assertEquals(expected, actual, 0d);
```

Comment:
Please set the allowed delta value as a parameter in the raster test base such as ` protected static final double FP_TOLERANCE = 1e--4`
`